ui/src/react-admin/modules/translations/translations.types.ts

```diff
@@ -23,4 +23,5 @@ export interface TranslationsOverviewV2Props extends DefaultProps {
 		onSave: () => void;
 		onClose: () => void;
 	}) => ReactNode;
+	isAvo?: boolean;
```

Comment:
Avoid using this boolean, it's already available using the AdminConfigManager

Reply:
```
AdminConfigManager.getConfig().database.databaseApplicationType === AvoOrHetArchief.avo
```

Reply:
Done!